composer.json

```diff
@@ -32,6 +32,7 @@
         "doctrine/persistence": "^2.2",
         "psr/cache": "^1 || ^2 || ^3",
         "symfony/console": "^3.0 || ^4.0 || ^5.0 || ^6.0",
+        "symfony/polyfill-php72": "^1.23",
```

Comment:
This is in order to polyfill `spl_object_id`: https://github.com/symfony/polyfill-php72/blob/9a142215a36a3888e30d0a9eeea9766764e96976/Php72.php#L89-L100